dll/win32/netcfgx/tcpipconf_notify.c

```diff
@@ -2732,7 +2734,6 @@ LoadDNSSettings(
         dwSize = sizeof(DWORD);
         RegQueryValueExW(hKey, L"UseDomainNameDevolution", NULL, NULL, (LPBYTE)&This->pCurrentConfig->pDNS->UseDomainNameDevolution, &dwSize);
 
-        dwSize = 0;
```

Comment:
It's still good practice to do this, even though it's not technically necessary.

Reply:
The function takes `*lpcbData` as an input. We pass a buffer pointer that is not valid, so we should indicate that it has zero writable bytes in the parameter intended for this. The documentation for this function says that this is not necessary -- but passing a value of zero makes the code so obviously correct that you don't need to even look at the documentation.
Hundreds and hundreds of C functions work this way, and `NULL, 0` or `NULL, &variableWithZeroValue` will work for all of them.
If you find this confusing, that's an indication of your inexperience with common C programming paradigms, not an a problem with this code.

Reply:
P.S.:
Well, I assumed ReactOS would be a place where people would know/check how Windows functions work.
Anyway, I can understand your explanation and never said that was causing a bug.